README.md

```diff
@@ -711,6 +711,17 @@ const view = instance.views.getView('zoomview');
 view.setAmplitudeScale(1.0);
 ```
 
+## Cue events
+
+Emit events when the playhead reaches a point or segment boundary:
+
+```js
+const cues = new Peaks.CueEmitter(instance);
+cues.on('cue.point', function(id, time) { /* ... */ });
```

Comment:
I would prefer to use `points.enter`, `segments.enter`, `segments.exit` as event names, being more similar to the  TextTrackCue API.

Also, I think the handler functions should receive a `Point` or `Segment` object as parameter, rather than `id` and `time`.